boopmain.ino

```diff
@@ -117,12 +93,24 @@ void draw()
 void vibe_flash()
 {
     if(is_button_pressed) {
-        fill_solid(leds, NUM_LEDS, rainbowColors[color_select]); //If button is pressed, display colour from defined colour palette
-    }
+      for(int i = 0; i < NUM_LEDS; i++){
+		uint8_t bri = 255;
+		leds[i] = CHSV(global_hue, 200, bri);
+	}
+  }
     else {
-      fill_solid(leds, NUM_LEDS, 0x000000); //If no button press, LEDs off
+      for(int i = 0; i < NUM_LEDS; i++){
+      leds[i] = CHSV(0, 0, 0);
+      } //If no button press, LEDs off
```

Comment:
Not an issue but for future simplicity. You can set the value of `bri` to 255 if button down and 0 if not. 

Only the value of BRI determines if the pixels are on or off.